test/unit/vendor/lime/lime_harnessTest.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+require_once __DIR__.'/../../../bootstrap/unit.php';
+require_once __DIR__.'/tools/TestCase.php';
+require_once __DIR__.'/tools/lime_no_colorizer.php';
+
+class lime_harnessTest extends TestCase
+{
+    private function makeHarness(): lime_harness
+    {
+        $harness = new lime_harness();
+        $harness->output->colorizer = new lime_no_colorizer();
+
+        return $harness;
+    }
+
+    private function whenExecuteHarnessWithOneFileWillHaveStatusCodeAndOutput($name, $expectedOverallSucceed, $expectedOutput)
```

Comment:
The method name is not consistent with its content.

Need to choose between
- file or name
- status code or overall success